### ISSUE TITLE
numDatumsPerLabel does not return any results

### ISSUE THREAD

**Issue by loretoparisi** [stanfordnlp/CoreNLP#395]
When I start my [ColumnDataClassifier](https://nlp.stanford.edu/nlp/javadoc/javanlp/edu/stanford/nlp/classify/ColumnDataClassifier.html)

I can see how it calls `numDatumsPerLabel` method to print datum for each label:
```
22:08:19.128 [main] INFO  edu.stanford.nlp.classify.Dataset - numDatums: 8063
numDatumsPerLabel: {A=1116.0, B=221.0, C=2134.0, D=3551.0, E=1041.0}
numLabels: 5 [A, B, C, D, E]
```

But if I try to call this method from my class

```
 res = this.classifier.numDatumsPerLabel();
```

I get a {} object. In my understanding the [numDatumsPerLabel] method (https://nlp.stanford.edu/nlp/javadoc/javanlp/edu/stanford/nlp/classify/GeneralDataset.html#numDatumsPerLabel--) should return a [ClassicCounter](https://nlp.stanford.edu/nlp/javadoc/javanlp/edu/stanford/nlp/stats/ClassicCounter.html) object that calculates the increment for each labels.

I have my training and testing set loaded.

**Comment by J38** [comment]
To get that output you need to call `train.summaryStatistics();` on the `GeneralDataset` object representing the training data.  `numDatumsPerLabel()` is not a method of a classifier it is a method of a GeneralDataset.

**Comment by loretoparisi** [comment]
@J38 Thanks, yes sorry that one is a method of `GeneralDataset`. So regarding `summaryStatistics()` method when can I call it? I have my classifier loaded with the current saved model, and new and/or current training set/test set (depending if I'm going to do a new training or not). So having those two scenery:

1. loadClassifier property only
2. loadClassifier, trainFile, testFile properties

Will the `summaryStatistics()` work in both cases?

Thanks.

**Comment by J38** [comment]
A `ColumnDataClassifier` object has this method which will return a `Pair<GeneralDataset<String,String>, List<String[]>>` if you call this method `readAndReturnTrainingExamples(String fileName)`

You can get the first element of a Pair like this: `GeneralDataset<String,String> train = dataInfo.first();`
